pages/Landing/landing.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="https://ani-design.netlify.app/css/main.css" />
+		<link
+			rel="stylesheet"
+			href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+			integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+			crossorigin="anonymous"
+		/>
+    <link rel="stylesheet" href="../../css/common.css">
+		<link rel="stylesheet" href="./landing.css" />
+		<title>Ani Quizz</title>
+	</head>
+	<body>
+		<header class="quiz-header">
+			<p class="text-xhuge">Ani Quizz</p>
+			<a href="https://twitter.com/omkakatkar" class="text-huge text-blue"
+				><i class="fab fa-twitter"></i
+			></a>
+		</header>
+		<main class="flex-container quiz-main">
+			<div>
+				<h1 class="text-heading">Ani Quizz: The ultimate quizz app</h1>
+				<p class="text-huge">
+					Test your knowledge with our huge variety of quizes
+				</p>
+				<a href="../Home/home.html" class="btn bg-blue rounded quiz-hero-button"
+					>Lets Go!!</a
+				>
+			</div>
+			<div class="quiz-hero-container">
+				<img src="../../assets/hero.svg" alt="" />
```

Comment:
here also

Reply:
Thanks for the suggestion. I will add it.